src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_DRIVER =
+            "com.mysql.cj.jdbc.Driver";
+    private static final String DB_URL =
+            "jdbc:mysql://localhost:3306/jdbc";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Nikita13543");
+        try {
+            Class.forName(DB_DRIVER);
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't not load JDBC driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(DB_URL, DB_PROPERTIES);

```

Comment:
We don't need create DB_PROPERTIES. DriverManager has overloaded getConnection method that receive DB_URL, user and password
```suggestion
        return DriverManager.getConnection(DB_URL, user, pass);

```